src/exampleTypes/Styles.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+export default () => (
+  <style>
+    {`.contexture-search-box:focus {
+      background: rgba(255, 255, 255, 1);
```

Comment:
This is white, no? 🤔

Reply:
It is, but it's here to show the difference in these example styles just being about changing opacity. This file is pretty much throwaway code as a demo and isn't really intended to be used anywhere. In fact, this should probably get refactored to component-based overrides like the newer example types